var/spack/repos/builtin/packages/py-awscli/package.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAwscli(PythonPackage):
+    """This package provides a unified command line interface to
+       Amazon Web Services"""
+
+    homepage = "https://pypi.org/project/awscli/"
+    url      = "https://pypi.io/packages/source/a/awscli/awscli-1.16.111.tar.gz"
+
+    version('1.16.111', sha256='63c7900e65a76a1574e1d6e3aaa37c98403339a552428b47559d4e05081f8e4b')
+
+    depends_on('py-setuptools')
+    depends_on('py-docutils@0.10:')
+    depends_on('py-nose', type=('build', 'test'))
+    depends_on('py-tox@2.3.1:2.999', type='test')
```

Comment:
I don't think this dependency is actually required, probably only needed for version compatibility tests in Travis upstream.